src/utils.h

```diff
@@ -220,14 +221,33 @@ extern TSDLLEXPORT AttrNumber ts_map_attno(Oid src_rel, Oid dst_rel, AttrNumber
  * Return Oid for a schema-qualified relation.
  */
 static inline Oid
-ts_get_relation_relid(char const *schema_name, char const *relation_name, bool schema_missing_ok)
+ts_get_relation_relid(char const *schema_name, char const *relation_name, bool return_invalid)
 {
-	Oid schema_oid = get_namespace_oid(schema_name, schema_missing_ok);
+	Oid schema_oid = get_namespace_oid(schema_name, true);
 
 	if (OidIsValid(schema_oid))
-		return get_relname_relid(relation_name, schema_oid);
-
-	return InvalidOid;
+	{
+		Oid rel_oid = get_relname_relid(relation_name, schema_oid);
+
+		if (!return_invalid)
+			Ensure(OidIsValid(rel_oid),
+				   "relation id for chunk \"%s\".\"%s\" has invalid OID",
```

Comment:
make sense to change the message.

But I think
* `"%s.%s"` is invalid; as that would mean the schema is also part of the object name.
* ``%s.%s`` could be invalid as well in case one of them needs quotation - but in most cases okay
* `"%s"."%s"` is *ugly* for sure; but I think it will not have such issues

I [was using](https://github.com/timescale/timescaledb/pull/5701#discussion_r1206508864)  `quote_identifier` (which does automatic safety checking)  earlier.

Reply:
This is intended for and read by humans so "invalid" does not make sense and quoting is not necessary either. I think we should just stick to the conventions already in place for PostgreSQL code, for example [this case](https://github.com/postgres/postgres/blob/d64e6468f489effec356ce3501c0f226ac1cfcc0/src/backend/replication/logical/relation.c#L247):
```c
		ereport(ERROR,
				(errcode(ERRCODE_OBJECT_NOT_IN_PREREQUISITE_STATE),
				 errmsg_plural("logical replication target relation \"%s.%s\" is missing replicated column: %s",
							   "logical replication target relation \"%s.%s\" is missing replicated columns: %s",
							   missingattcnt,
							   remoterel->nspname,
							   remoterel->relname,
							   missingattsbuf.data)));
```